salt/base/swap.sls

```diff
@@ -0,0 +1,16 @@
+{% set swapfile = salt["pillar.get"]("swapfile", {}) %}
+{% set size = swapfile.get("size", "1024") %}
+{% set path = swapfile.get("path", "/swapfile") %}
```

Comment:
```suggestion
{% set path = swapfile.get("path") %}
```

I'm not sure want to _always_ add this (really thinking mostly in dev where it is almost never necessary), so give a way to opt out, see additional suggested change.